packages/strapi-helper-plugin/lib/src/components/PopUpWarning/index.js

```diff
@@ -39,24 +42,20 @@ function PopUpWarning({
 }) {
   const buttons = [
     {
-      className: 'secondary',
-      id: 'ctaCancel',
+      color: 'cancel',
       handleClick: toggleModal,
       message: content.cancel || 'components.popUpWarning.button.cancel',
-      style: {},
     },
     {
-      className: 'primary',
-      id: 'ctaConfirm',
+      color: 'delete',
       handleClick: onConfirm,
       message: content.confirm || 'components.popUpWarning.button.confirm',
-      style: {},
     },
   ];
+
   const singleButton = [
     {
-      className: 'primary',
-      id: 'ctaConfirm',
+      color: 'delete',
       handleClick: onConfirm,
       message: content.confirm || 'components.popUpWarning.button.confirm',
       style: { width: '100%' },
```

Comment:
Maybe the single button can set a `fullWidth` prop to true. We must avoid inline style customization to keep a standard style format.

Reply:
I'm okay with you that it's not the best way to change width but there are the props we pass directly to a Buffet compo. 

Most of the time they just fit their content but some times we set custom style (not only a width to 100% but also different left and right padding cause we want them to be larger respecting the translations).

We should check and update many buttons on Strapi to enable different custom buttons.

For now, we don't exactly know which standards these buttons will have (design system is ongoing) but this work is in progress and it will help us to generalize the entire app.